internal/http/interceptors/auth/signed_url/loader/loader.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2025 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+package loader
+
+import (
+	// Load core authentication strategies.
+	_ "github.com/cs3org/reva/v3/internal/http/interceptors/auth/signed_url/strategy/signed_url"
+	// Add your own here.
```

Comment:
leftover?

Reply:
Well no, this comment is there in all the `loader`s: if you create a new strategy, you have to place it here so it gets `init`ed